index.html

```diff
@@ -14,9 +14,9 @@
         
         <div class="images">
          <div class="space">
-            <div class="image01"><img src="assets/Shape.png" alt="#" class="shape00"></div>
+            <!-- <div class="image01"><img src="assets/Shape.png" alt="#" class="shape00"></div>
             <div class="image02"><img src="assets/Shape_1.png" alt="#"  class="shape01"></div>
```

Comment:
- [OPTIONAL ]
Kindly try to remove useless comments in your code that will make your code more readable 😃